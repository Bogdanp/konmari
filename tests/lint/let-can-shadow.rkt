#lang racket/base

(define (a-page req)
  (let loop ([req req])
    (send/suspend/dispatch
     (lambda (embed/url)
       (displayln req)
       (embed/url loop)))))
