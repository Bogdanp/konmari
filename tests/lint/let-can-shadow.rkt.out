/let-can-shadow.rkt:3:10:warning:identifier 'a-page' is never used
